### PR TITLE
Docs reorg for new scale section, amongst other things

### PR DESCRIPTION
**Pull request by ChrisChinchilla**

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
Quite a lot of changes here to review, you've seen a lot of them already. Note that the 'install/index' page is currently just dummy text, I'm not sure what to do with 'docs/install' vs 'download' pages, they duplicate at the moment, but page layouts are different.

**Comment by ChrisChinchilla** [comment]
@joemoe Anyone else who wants to look too.

File: pages/docs/connect/index.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title: Connect to a Cluster
+author: Chris Ward
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ab hoc autem quaedam non melius quam veteres, quaedam omnino relicta. Vestri haec verecundius, illi fortasse constantius. Est, ut dicis, inquit; Restatis igitur vos; Universa enim illorum ratione cum tota vestra confligendum puto. Duo Reges: constructio interrete. Et quidem iure fortasse, sed tamen non gravissimum est testimonium multitudinis. Quae cum dixisset paulumque institisset, Quid est?
```
Comment (joemoe) [inline pages/docs/connect/index.html]: What are we doing with this dummy text. This should be ready, when we publish it right?

File: pages/docs/scale/0_to_50_nodes.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Try running a simple query on the cluster, such as:
+
+```sql
+SELECT count(*) AS quantity, user['verified']
+FROM tweets
+GROUP BY user['verified']
+ORDER BY quantity DESC  limit 100;
+```
+
+Add and remove nodes to the cluster and try issuing the same query each time, you should notice no change in the result and very little difference in speed.
```
Comment (joemoe) [inline pages/docs/scale/0_to_50_nodes.html]: is this the only new article?

File: pages/docs/scale/multi_node_setup.html [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+# port accessible from outside
+transport.publish_port: 4321
+```
+<div class="admonition seealso">
+<p class="first admonition-title">Related</p>
+<ul>
+<li><a href="https://crate.io/docs/reference/configuration.html#conf-hosts">Host settings for Nodes</a></li>
+<li><a href="https://crate.io/docs/reference/configuration.html#conf-ports">Host settings for Ports</a></li>
+</ul>
+</div>
```
Comment (joemoe) [inline pages/docs/scale/multi_node_setup.html]: this existed already in the crate docs?

File: pages/docs/scale/multi_zone_setup.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+```yaml
+cluster.routing.allocation.awareness.force.zone.values: us-east-1,us-west-1
+```
+
+When set, no more replicas than needed are allocated on a specific group of nodes.
+
+**An example**: You have 2 nodes with `zone` set to `us-east-1` and create a table with 8 shards and 1 replica. 8 primary shards will be allocated and the 8 replica shards left unassigned. Only when you add a new node with `zone` set to `us-west-1` will the replica shards be allocated.
+
+By using these settings and their mechanisms correctly, you should be able to setup a cluster that spans across multiple zones and fulfills the 3 requirements above, but be aware of the drawbacks such a setup can have.
```
Comment (joemoe) [inline pages/docs/scale/multi_zone_setup.html]: this too?

File: static/css/crateio-rtd.css [hunk 5]
```diff
@@ -39,102 +39,121 @@ blockquote {
...
 }
 .wrapper-navleft .bs-docs-sidebar {
-  border: 1px solid #e7e7e7;
+	border: 1px solid #e7e7e7;
 }
 .wrapper-content-right li {
-    font-size: 16px;
-    line-height: 1.5em;
+	font-size: 16px;
+	line-height: 1.5em;
```
Comment (joemoe) [inline static/css/crateio-rtd.css]: there are a lot of indentation changes. we should make sure to avoid those.

**Pull request merged**